### PR TITLE
feat(keebwerk): added VIA support keymap for keebwerk nano slider

### PR DESCRIPTION
**Pull request by wilfrid-li**
## Description

Added VIA support for keebwerk nano slider
Fixed midi2vol keymap where comma symbol is missing from enum "custom_layers"

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebwerk/nano_slider/keebwerk_nano_via.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (zvecr) [inline keyboards/keebwerk/nano_slider/keebwerk_nano_via.json:1]: This file does not belong in this repo and should be removed.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/keebwerk/nano_slider/keebwerk_nano_via.json (hunk 1 above)
Comment (wilfrid-li) [reply keyboards/keebwerk/nano_slider/keebwerk_nano_via.json]: New commit made to remove the JSON file

**Review by drashna — APPROVED** [review]

File: keyboards/keebwerk/nano_slider/keymaps/via/readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# The default keymap for nano_slider
```
Comment (noroadsleft) [inline keyboards/keebwerk/nano_slider/keymaps/via/readme.md:1]: This file can be deleted.

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**